docs/release-notes/NuGet-6.12.md

```diff
@@ -17,6 +15,147 @@ NuGet distribution vehicles:
 
 <sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
 
-## Not yet released
+## Known Issues
+
+* Project and package in the same graph with the same name but different dependencies may lead to incorrect versions of the dependencies of that id [#13888](https://github.com/NuGet/Home/issues/13888)
+* VS PM UI shows warning icon about package vulnerability even after upgrade [#13866](https://github.com/NuGet/Home/issues/13866)
+* dotnet nuget why reports missing argument, even though it ran [#13908](https://github.com/NuGet/Home/issues/13908)
+
+## Summary: What's New in 6.12.1
+
+NuGet 6.12.1 is available in Visual Studio 17.12.0 and the .NET 9.0.101 SDK.
+
+### Issues fixed in this release
+
+* Deserializing an empty version range in a package dependency fails in .NET SDK 9.0.100-rc.2 [#13869](https://github.com/NuGet/Home/issues/13869)
+
+## Summary: What's New in 6.12
+
+NuGet 6.12.0 is available in the .NET 9.0.100 SDK.
+
+* NuGet's dependency graph resolution algorithm does not scale well for large graphs - [#13692](https://github.com/NuGet/Home/issues/13692)
```

Comment:
Not sure if I commented on this originally, but It's probably ok to take creative freedom and write it in a way that's nicer. 

```suggestion
* Introduce optimized dependency resolver that scales well for large graphs - [#13692](https://github.com/NuGet/Home/issues/13692)
* ```

Reply:
> take creative freedom

ok!

```suggestion
* The hypotenuse of the monad is tangential to the flux capacitor second derivitive  - [#13692](https://github.com/NuGet/Home/issues/13692)
```